qdev_wrappers/T3/customised_instruments.py

```diff
@@ -332,31 +250,78 @@ def _set_fine_voltage(self, voltage):
         self.volt.set(coarse_part)
         slot.channels[fine_chan].volt.set(fine_scaled)
 
-    def set_all(self, voltage_value, set_dcbias=False):
```

Comment:
@ThorvaldLarsen  do we want to remove this?

Reply:
It is some old stuff that we never used. Lets remove it for now and get a clean driver to start from. Then maybe later there might be a few functions we want to add again.